orchestrator/lib/coolfhir/util.go

```diff
@@ -64,6 +65,57 @@ func ValidateLogicalReference(reference *fhir.Reference, expectedType string, ex
 	return nil
 }
 
+// ValidateCareTeamParticipantPeriod validates that a CareTeamParticipant has a start date, and that the start date is in the past
+// end date is not required, but if present it will validate that it is in the future
+func ValidateCareTeamParticipantPeriod(participant fhir.CareTeamParticipant) error {
```

Comment:
provide `now` in function signatures, eases testing an makes function applicable in more instances

Reply:
Yes that's a good call, will add it